lib/rspecq/queue.rb

```diff
@@ -135,12 +135,14 @@ def requeue_job(job, max_requeues)
       )
     end
 
-    def save_rerun_command(job, cmd)
-      @redis.hset(key("job_metadata"), job, cmd)
+    # General purpose method.
+    # Used to save metadata regarding a job, i.e its backtrace.
+    def save_job_metadata(job, key, value)
+      @redis.hset(key("job_metadata"), "#{job}_#{key}", value)
     end
 
-    def rerun_command(job)
-      @redis.hget(key("job_metadata"), job)
+    def job_metadata(job, key)
+      @redis.hget(key("job_metadata"), "#{job}_#{key}")
```

Comment:
Different builds might have different example IDs and different flaky tests. However lobal "metadata" key for all builds here, which means that results from different builds will be mingled.

Is that intentional?